Base/src/main/java/io/deephaven/base/string/EncodingInfo.java

```diff
@@ -56,7 +56,8 @@ public String getEncodingName() {
     }
 
     /**
-     * Can this encoding info's charset be encoded or decoded by simple linear assignment of char->byte or byte->char.
+     * Can this encoding info's charset be encoded or decoded by simple linear assignment of char-&gt;byte or
+     * byte-&gt;char.
```

Comment:
Should these be using HTML arrows instead (e.g. `&larr;` or `&rarr`)?

Reply:
That's really up to the author - I'm making it not-wrong, not necessarily reading the author's mind for what they originally intended.

Reply:
I would not hold up for any of these arrow comments.  These can be handled later.